deploy/examples/ingress-with-annotations.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: jaegertracing.io/v1
+kind: Jaeger
+metadata:
+  name: jaeger
+spec:
+  strategy: allInOne 
+  storage:
+    type: memory 
+    options: 
+      memory: 
+        max-traces: 1000000
+  ingress:
+    enabled: true
+    annotations:
+      kubernetes.io/ingress.class: nginx
```

Comment:
Is this needed?

Reply:
I was thinking to show how to use ingress annotations here

Reply:
Then we can have two examples, each with one use-case? Unless all the people looking for an example of how to set the annotation will also need to set the host, and vice-versa.